src/zarate.jl

```diff
@@ -105,8 +105,7 @@ function ordering_problem(layout::Zarate, graph, layer2nodes)
     m = Model(layout.ordering_solver)
     set_silent(m)
 
-    T = JuMP.Containers.DenseAxisArray{VariableRef,1,Tuple{Vector{Int64}},Tuple{Dict{Int64,Int64}}}
-    node_is_before = Vector{T}(undef, nv(graph))
+    node_is_before = Vector{Any}(undef, nv(graph))
```

Comment:
I tested this configuration and it works; even if it is slightly less efficient, it is much more flexible for possible changes in the JuMP implementation of DenseAxisArray